docs/content/concepts/code-locations/workspace-files.mdx

```diff
@@ -221,7 +221,7 @@ If you're using the built-in [Helm chart](/deployment/guides/kubernetes/deployin
 
 ### Loading relative imports
 
-By default, code is loaded with Dagit's working directory as the base path to resolve any local imports in your code. Using the `working_directory` key, you can specify a custom working directory for relative imports. For example:
+By default, code is loaded with `dagster-webserver`'s working directory as the base path to resolve any local imports in your code. Using the `working_directory` key, you can specify a custom working directory for relative imports. For example:
```

Comment:
I went with `dagster-webserver` purposefully here since "working directory" is clearer in the context of an executing command, and `dagster-webserver` is the command that you run.